CONTRIBUTING.md

```diff
@@ -239,6 +239,22 @@ Before submitting a pull request, please make sure that:
 3. Comments are rendered correctly in the `task godocs` output.
 4. `task all` passes.
 
+**For creating and submitting a pull request, please follow these guides:**
+
+1. Please create a **draft pull request** when you start working on an issue.
+   Once it’s ready for review, mark it as such.
+   You don’t need to switch between draft and non-draft states after that
```

Comment:
I think that point could be either removed or reformulated. We created draft PRs mostly to track our work. External contributors may not be comfortable with creating "unready" PRs.

Reply:
Ok, removed.